keyboards/wekey/we27/keymaps/via/keymap.c

```diff
@@ -0,0 +1,98 @@
+/* Copyright 2021 @wekey
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#ifdef ENCODER_ENABLE
```

Comment:
And here, since ENCODER_ENABLE cannot be changed by VIA itself - it will always be enabled since you have it set as `yes` in the keyboard-level `rules.mk`.